dora/core/client/fs/src/main/java/alluxio/client/file/PositionReadFileInStream.java

```diff
@@ -38,8 +41,18 @@ public class PositionReadFileInStream extends FileInStream {
   private boolean mClosed;
   private final PositionReader mPositionReader;
   private final PrefetchCache mCache;
+  private final URIStatus mURIStatus;
+  private final DoraCacheClient mClient;
+  // Preload requests are async so a cached thread pool is used here.
+  private final Executor mPreloadThreadPool = Executors.newCachedThreadPool();
```

Comment:
there's mRpcFutureStub in DefaultBlockWorkerClient so we can just leverage future stub instead of creating new threadpool